protocol/src/main/scala/lsug/protocol/protocol.scala

```diff
@@ -216,14 +226,15 @@ object Speaker {
   case class Profile(
       id: Id,
       name: String,
-      photo: Option[Asset]
+      photo: Option[Asset],
+      pronoun: Option[Pronoun]
   )
 
   object Profile {
     implicit val codec: Codec[Profile] = deriveCodec[Profile]
     implicit val eq: Eq[Profile] = Eq.instance {
-      case (Profile(i, n, p), Profile(ii, nn, pp)) =>
-        i === ii && n === nn && p === pp
+      case (Profile(i, n, p, q), Profile(ii, nn, pp, qq)) =>
```

Comment:
pronoun obviously doesn't start with `q` but `p` was already taken here - what would your convention be instead?

Reply:
I'm not particularly fussy with these things.  Personally, I tend to use full words for anything with meaning, and keep to letters for abstract types.
For example, this would always be `fa` and `f`, because there's nothing more concrete it could mean:

```scala
def map(fa: F[A])(f: A => B): F[B] = ...
```

I'd also use a short form if the type was clearly inferred, like:
```scala
val ns: List[SpeakerName] = ???
```

But if the type was ambiguous, I'd use a full word:
```scala
val speakerNames: List[String] = ???
```

This is entirely a matter of personal preference as everyone finds different things easier to read.  There's no fixed convention around it.

In this case, what you have is pretty clear.